torch/_functorch/_aot_autograd/graph_capture.py

```diff
@@ -312,6 +312,9 @@ def aot_dispatch_base_graph(
                 include_stride=True,
                 include_device=True,
                 colored=True,
+                # For more expanded output (but can't default to this because it
+                # affects tests):
+                # expanded_def=True
```

Comment:
delete? or do you want to just pass `expanded_def=False` to remember this flag exists?

Reply:
I'd like to keep it just because it's a useful flag to remember it exists - maybe I'll put it behind a envvar (I'll do that as a follow-up).